xc7/utils/prjxray_routing_import.py

```diff
@@ -230,6 +231,28 @@ def check_feature(feature):
 
         return ' '.join((feature, enable_buffer_feature))
 
+    if CLK_HROW_CK_MUX_REGEX.fullmatch(feature_path[-1]):
+        bufhce_loc_string = feature_path[-1].split('_')[-1]
+        x_loc_str = bufhce_loc_string[0]
+        if 'L' in x_loc_str:
+            x_loc = 0
+        elif 'R' in x_loc_str:
+            x_loc = 1
+        else:
+            assert False, "Impossible to determine X location of BUFHCE"
+
+        y_loc = bufhce_loc_string[1:]
+        bufhce_loc = 'BUFHCE_X{}Y{}'.format(x_loc, y_loc)
```

Comment:
This code only works for the lowest block of BUFHCE, because the y_loc increases on the higher CMTs, but the segbits is modulo the height of the number of BUFHCE's (e.g. 12).  The higher ones should by y_loc % 12, but let's build that table from the connection database.  So please add something like `populate_bufg_rebuf_map` or `populate_hclk_cmt_tiles` to create a lookup.